server/datastore/mysql/migrations/tables/20211102135149_AddHostSeenTimesTable.go

```diff
@@ -0,0 +1,39 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20211102135149, Down_20211102135149)
+}
+
+func Up_20211102135149(tx *sql.Tx) error {
```

Comment:
For posterity, I believe it makes sense to either link an issue or briefly explain why we are doing this.